src/System.Net.Sockets/ref/System.Net.Sockets.csproj

```diff
@@ -1,22 +1,17 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <ProjectGuid>{834E3534-6A11-4A8D-923F-35C1E71CCEC3}</ProjectGuid>
-    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'netfx'">true</IsPartialFacadeAssembly>
     <Configurations>netcoreapp-Debug;netcoreapp-Release;uap-Debug;uap-Release</Configurations>
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Net.Sockets.cs" />
-  </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' != 'netfx'">
     <Compile Include="System.Net.Sockets.netcoreapp.cs" />
+  </ItemGroup>
+  <ItemGroup>
```

Comment:
NIT: No need for an extra `<ItemGroup>` here, since we can simply use the Compile one for ProjectReference as well

Reply:
I must disagree. We keep Compile items separate from ProjectReferences in corefx.

Reply:
yup you are right, didn't notice that it was ProjectReferences on the other ItemGroup.